cli/flags.rs

```diff
@@ -63,6 +63,8 @@ impl Default for DenoSubcommand {
 
 #[derive(Clone, Debug, PartialEq, Default)]
 pub struct DenoFlags {
+  /// Vector of CLI arguments - these are user script arguments, all Deno
+  /// specific flags are removed.
```

Comment:
No longer true? `deno run --allow-net foo.ts --allow-run --allow-write` should give `argv == vec!["foo.ts", "--allow-run", "--allow-write"]`.

Reply:
That’s right but I think the comment is still more or less correct?